src/dos/cdrom_image.cpp

```diff
@@ -70,80 +95,148 @@ int CDROM_Interface_Image::BinaryFile::getLength()
 	return length;
 }
 
-#if defined(C_SDL_SOUND)
+Bit16u CDROM_Interface_Image::BinaryFile::getEndian()
+{
+	// Image files are read into native-endian byte-order
+	#if defined(WORDS_BIGENDIAN)
```

Comment:
I *think* the style is to move all preprocessor directives to column 0 (not 100% sure though).

Reply:
Thanks, will make this change (and grep for anywhere else it might happen)

Reply:
They're all moved now.